Src/Common/CrossComponentCorrelationEventSource.cs

```diff
@@ -15,12 +15,12 @@
     /// <summary>
     /// ETW EventSource tracing class.
     /// </summary>
-    [EventSource(Name = "Microsoft-ApplicationInsights-Extensibility-CrossComponentCorrelation")]
-    internal sealed class CrossComponentCorrelationEventSource : EventSource
+    [EventSource(Name = "Microsoft-ApplicationInsights-Extensibility-AppMapCorrelation")]
+    internal sealed class AppMapCorrelationEventSource : EventSource
```

Comment:
Why rename? Can you please add it to the list of event sources in functional tests